paystakk/api.py

```diff
@@ -166,31 +173,56 @@ def __getattr__(self, item):
 
     @property
     def ctx(self):
+        """
```

Comment:
Properties don't need docstrings. Anybody looking at a property already knows what it is doing